Composer/packages/client/src/pages/botProject/create-publish-profile/PublishProfileDialog.tsx

```diff
@@ -89,10 +96,7 @@ export const PublishProfileDialog: React.FC<PublishProfileDialogProps> = (props)
     PluginAPI.publish.closeDialog = closeDialog;
     PluginAPI.publish.onBack = () => {
       setPage(Page.ProfileForm);
-      setTitle({
-        title: current ? formatMessage('Edit publishing profile') : formatMessage('Add new publishing profile'),
-        subText: formatMessage('A publishing profile provides the secure connectivity required to publish your bot. '),
-      });
+      setTitle(formatDialogTitle(current));
```

Comment:
same comment L70

Reply:
Will need to pick this up in a future PR.